_includes/templates/summaries/provide-feedback.html

```diff
@@ -0,0 +1,18 @@
+<h2 class="h4">Provide Feedback</h2>
+<p class="short-desc">
+    We take pride in our feedback-centered culture
+    that helps us recruit great people.
+    If you have any feedback, or any specific questions, let us know at.
+</p>
+<p class="short-desc">
+    <ul class="list list__icons">
+        <li class="list_item">
+            <span class="cf-icon cf-icon-email list_icon"></span>
+            <span class="h5 list_text">Email</span>
+            <p><a href="mailto:jobs@consumerfinance.gov"
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/fe406bb5534f5de096b88ca71e04f7729d656452